pipeline.py

```diff
@@ -51,19 +61,26 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self._storage = {}
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        paths_to_files = list(Path(self.path_to_raw_txt_data).glob('*_raw.txt'))
```

Comment:
why do you create a list - instead of directly iterating through a glob result? that would be much less memory consuming